<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>790</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="14">
     <widget class="QPushButton" name="Btn_recover">
      <property name="text">
       <string>水平/垂直恢复</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="13">
     <widget class="QLineEdit" name="lineEdit_cfgpath">
      <property name="text">
       <string>C:/Users/Administrator/Desktop/COMTRADE/COMTRADE/BAY01_0088_20190428_140316_668.cfg</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_show">
      <property name="text">
       <string>show</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>290</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>保存显示图片</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="13">
     <widget class="QPushButton" name="Btn_hscale">
      <property name="text">
       <string>水平压缩</string>
      </property>
     </widget>
    </item>
    <item row="2" column="12">
     <widget class="QPushButton" name="Btn_hscale0">
      <property name="text">
       <string>水平拉伸</string>
      </property>
     </widget>
    </item>
    <item row="2" column="15">
     <widget class="QPushButton" name="Btn_showall">
      <property name="text">
       <string>水平/垂直全显</string>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QPushButton" name="Btn_vscale0">
      <property name="text">
       <string>垂直拉伸</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="13">
     <widget class="QLineEdit" name="lineEdit_picturepath">
      <property name="text">
       <string>C:\wave\customplot.png</string>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QPushButton" name="Btn_vscale1">
      <property name="text">
       <string>垂直压缩</string>
      </property>
     </widget>
    </item>
    <item row="1" column="14" colspan="2">
     <widget class="QPushButton" name="btn_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="16">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="Btn_T0">
      <property name="text">
       <string>故障点</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>黑色</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>白色</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄色</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>红色</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>绿色</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="Btn_max">
      <property name="text">
       <string>极大值</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>背景色：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="Btn_min">
      <property name="text">
       <string>极小值</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>谐波分析</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>相角分析</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="14">
     <widget class="QPushButton" name="btn_pathselect">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="15">
     <widget class="QPushButton" name="btn_back">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
